### PR TITLE
add SNO_FARCONNECT to the help text

### PR DESCRIPTION
**Pull request by xfnw**
snomask +F seems to be missing from the help text

### CODE REVIEW

**Comment by examknow** [comment]
We should probably also mention that this requires `extensions/sno_farconnect` to be loaded.

**Comment by xfnw** [comment]
hmm, the other entries dont mention required extensions

**Comment by aaronmdjones** [comment]
Follow the convention in the `UMODE` help, and put a `?` next to snomasks that require extensions?

File: help/opers/snomask [hunk 1]
```diff
@@ -1,6 +1,8 @@
 MODE <nick> +s <+|-><modes>
 
-Server notice masks:
+Server notice masks: (? designates that the umode is provided
```
Comment (aaronmdjones) [inline help/opers/snomask:3]: snomask, not umode.

**Review by aaronmdjones — CHANGES_REQUESTED** [review]

**Review by aaronmdjones — APPROVED** [review]

**Pull request merged**